src/main/java/module/matches/TeamsRatingPanel.java

```diff
@@ -98,14 +97,15 @@ private void addListeners() {
 
 	private void initComponents() {
 
+		var fontSize = UserParameter.instance().fontSize;
 
 		f =  new JLabel("").getFont();
 		f = f.deriveFont(f.getStyle() | Font.BOLD);
 
 		bars = new JProgressBar[14];
 		for (int i = 0; i < 14; i++) {
 			bars[i] = new JProgressBar(0, 100);
-			bars[i].setPreferredSize(new Dimension(200, 20)); //25 if nimbus
+			bars[i].setPreferredSize(new Dimension(20 * fontSize, 2 * fontSize)); //25 if nimbus
```

Comment:
Am I right in saying the comment at the end of the line is no longer valid/relevant?

Reply:
no idea, what is the reason for that comment. i will remove it.